### ISSUE TITLE
NativeStorage undefined in browser

### ISSUE THREAD

**Issue by glewiswp** [TheCocoaProject/cordova-plugin-nativestorage#100]
I can't seem to get NativeStorage working in Chrome browser for testing. It works fine in the ios emulator but not in the browser. It is rather tedious to have to keep transpiling the application and re-booting the ios simulator for testing. 

```js
var ns = (typeof NativeStorage == "undefined") ? false : true;
if( ns ) {
   alert( 'Active' );
} else {
   alert( 'Inactive' );
}
```

Returns inactive in browser.

Chrome:
`Version 61.0.3163.100 (Official Build) (64-bit)`

NativeStorage installed via:
```bash
$ cordova plugin add https://github.com/TheCocoaProject/cordova-plugin-nativestorage
```
Out of `$ cordova plugin ls`:
```bash
cordova-plugin-nativestorage 2.2.2 "NativeStorage"
```

Working with:
```bash
$ cordova emulate ios
```

Doesn't work with:
```bash
$ cordova run browser
```

**Comment by alokrajiv** [comment]
The mission-critical tests are passing... We are aware of some error functions misbehaving on the browser but not anything critical:

<img width="459" alt="screen shot 2017-12-05 at 10 58 21 am" src="https://user-images.githubusercontent.com/8435139/33587986-86a7a734-d9ab-11e7-9c41-d14d8d567651.png">

**Comment by alokrajiv** [comment]
Closing because of in-activity.

Not an plugin issue from my understanding.

**Comment by GillesC** [comment]
Fixed in commit 27be91e8acb759fd2ae8e8a8b73e725b1a0a84fc
